client.go

```diff
@@ -67,6 +67,8 @@ type Client interface {
 }
 
 const (
+	// Default ClientID
+	DefaultClientID = "sarama"
```

Comment:
There's no reason to make this a public constant.

Also, while I understand the intent behind putting it in `client.go` I think it makes more sense in `config.go`.